src/style.css

```diff
@@ -1 +1,176 @@
-/* empty source */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: fit-content;
+  padding: 16px;
+
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 16px auto 0;
+}
+
+.card__text {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin: 0;
+}
+
+.card__text--1 {
+  width: 166px;
+  height: 36px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+
+  color: #060b35;
+}
+
+.card__text--2 {
+  height: 14px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-top: 4px;
+
+  color: #616070;
+}
+
+.card__rate {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 16px;
+  width: 166px;
+}
+
+.stars {
+  display: flex;
+  line-height: 14px;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(/images/star.svg);
```

Comment:
```suggestion
  background-image: url(./images/star.svg);
```

Reply:
i think thats why the stars doesn't appear.

Reply:
In demo link I can see the stars! Don't they appear for you?

Reply:
hmm wierd, first time they doesn't appeaared, i see them now.

Reply:
good :)